PacketDotNet/ARPPacket.cs

```diff
@@ -21,7 +21,7 @@
 using System.Collections.Generic;
 using System.Net.NetworkInformation;
 using System.Text;
-using MiscUtil.Conversion;
+using PacketDotNet.MiscUtil.Conversion;
```

Comment:
Not opposed to the change but I'd like to hear the argument for folding this in vs. leaving it in its own namespace.